tensorflow_lite_support/c/task/vision/image_classifier.cc

```diff
@@ -88,6 +88,17 @@ StatusOr<ImageClassifierOptionsCpp> CreateImageClassifierCppOptionsFromCOptions(
   cpp_options.set_score_threshold(
       c_options->classification_options.score_threshold);
 
+  cpp_options.mutable_base_options()
```

Comment:
Hardcoding core ml delegate settings here since these changes are only for testing on a device.

Reply:
Once issues are fixed these can be implemented in the iOS framework.